### PR TITLE
Add pipeline for corp transfer service

### PR DESCRIPTION
**Pull request by mikelapierre**

### CODE REVIEW

**Comment by raykao** [comment]
@lynn-orrell - can you review this?  Would like to make the xfer service func to be dynamic for the reddog-hybrid-arc deployment RE: this issue on hybrid-arc https://github.com/Azure/reddog-hybrid-arc/issues/45

File: .github/workflows/package-corp-transfer-service.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      image: ${{ steps.main.outputs.image }}
+      repository: ${{ steps.main.outputs.repository }}  
+    steps:
+      - id: main
+        run: |
+          echo ::set-output name=version::$(echo ${GITHUB_SHA} | cut -c1-7)
+          echo ::set-output name=created::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+          echo ::set-output name=project::CorporateTransferService
+          echo ::set-output name=image::reddog-retail-corp-transfer-service
+          echo ::set-output name=repository::ghcr.io/mikelapierre/reddog-code
```
[the remark this answers is not in the document]
Comment (mikelapierre) [reply .github/workflows/package-corp-transfer-service.yaml]: No reason, just forget to rename it. I've fixed it in the latest commit.

**Review by lynn-orrell — APPROVED** [review]
lgtm

**Comment by mikelapierre** [comment]
@lynn-orrell  - can you start the action to generate the container image please?

**Pull request merged**